.github/workflows/ci.yml

```
@@ -19,10 +19,16 @@ jobs:
       fail-fast: false
 
       matrix:
-        ruby: [2.5.8, 2.6.6, 2.7.2, 3.0.0, jruby-9.2.16.0]
+        deps: [rails_61, rails_70]
+        ruby: [2.5.9, 2.6.9, 2.7.5, 3.0.3, jruby-9.3.2.0]
```

We list the Ruby versions first.

```suggestion
        ruby: [2.5.9, 2.6.9, 2.7.5, 3.0.3, jruby-9.3.2.0]
        deps: [rails_61, rails_70]
```

Thank you for updating these! If you want to do the same for InheritedResources, I'll approve that for you.

---

Done at activeadmin/inherited_resources#708

---

Thanks!